pantheon-bundle/frontend/src/app/searchFilter.tsx

```diff
@@ -47,6 +47,7 @@ class SearchFilter extends Component<any, any> {
 
     const moduleTypeItems = [
```

Comment:
can we have a more generic name i.e. instead of moduleTypeItems like contentTypeItems ?